.github/workflows/ci.yml

```diff
@@ -95,29 +95,51 @@ jobs:
 
   core-test:
     needs: prep-testbed
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest]
```

Comment:
I would recommend to add windows-latest as well in CI

Reply:
windows is added below.

Reply:
why not included in `matrix.os`

Reply:
Because of the matrix `${{test_paths}}`, windows test will also have three parts: `tests/`, `tests/docarray/`, `tests/executor/`, repeat testing it not necessary.